.github/workflows/validate.yml

```diff
@@ -52,6 +52,14 @@ jobs:
           path: ${{ steps.set-puppeteer-cache.outputs.PUPPETEER_CACHE_DIR }}
           key: puppeteer-${{ runner.os }}-${{ hashFiles('./pnpm-lock.yaml') }}
 
+      - name: Cache babel-loader
+        id: babel-loader-cache
+        uses: actions/cache@v3
+        with:
+          path: 'fixtures/*/node_modules/.cache/babel-loader'
+          key: babel-loader-${{ runner.os }}-${{ hashFiles('./pnpm-lock.yaml') }}
+          restore-keys: babel-loader-${{ runner.os }}-
```

Comment:
Just checking, is the trailing "-" correct?

Reply:
All [these examples](https://github.com/actions/cache/blob/0c45773b623bea8c8e75f6c82b208c3cf94ea4f9/examples.md#c---nuget) have a trailing hyphen, so I'd say yes.

It's probably not required, I think the restore key is just meant to be a non-unique prefix of the `key`, so it would probably work without the `-` too.